addon/ajax-request.js

```diff
@@ -160,6 +160,15 @@ export default class AjaxRequest {
   }
 
   /**
+   * calls `request()` but forces `options.type` to `DELETE`
+   * alias for `del()`
+   * @public
+   */
+  delete(...args) {
+    return this.del(...args);
```

Comment:
... and change this to `...arguments` instead, since `arguments` will always be defined as an array-like object of all arguments passed to the function.